pkg/cryptoauthlib/Kconfig

```diff
@@ -0,0 +1,40 @@
+# Copyright (c) 2022 Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config PACKAGE_CRYPTOAUTHLIB
+    bool "Microchip CryptoAuthentication Library package"
+    depends on TEST_KCONFIG
+    select MODULE_AUTO_INIT_SECURITY
+    select MODULE_CRYPTOAUTHLIB_CONTRIB
+
+config MODULE_CRYPTOAUTHLIB_CONTRIB
+    bool
+    depends on TEST_KCONFIG
+    select MODULE_ZTIMER
+    select MODULE_ZTIMER_USEC
+    select MODULE_ZTIMER_PERIPH_TIMER
```

Comment:
```suggestion
    select ZTIMER_USEC
```
Here you can just use this

Reply:
What about this suggestion?

Reply:
That doesn't work (I tried): there were missing ztimer dependencies on some platform (comparing the output of info-modules with and without Kconfig).

Reply:
Ah, it's `ZTIMER_USEC`, not `MODULE_ZTIMER_USEC`. I think I tried the latter.

Reply:
Yeah, it is a workaround to avoid a circular dependency related to xtimer and ztimer. Hopefully will go away once we switch over

Reply:
Ok, I pushed that suggestion as well. And squashed.